src/ucs/config/parser.c

```diff
@@ -1339,10 +1339,9 @@ UCS_STATIC_INIT {
 }
 
 UCS_STATIC_CLEANUP {
-    int UCS_V_UNUSED dummy;
     const char *key;
 
-    kh_foreach(&ucs_config_parser_env_vars, key, dummy, {
+    kh_foreach_key(&ucs_config_parser_env_vars, key, {
```

Comment:
```kh_foreach``` could not be used here because ```val``` is not initialized in hash which cause crash if there is at least one key